src/subgraph-client/subgraph-client.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { gql, request } from 'graphql-request';
+import { TokensUnwrappedsResponse } from '../subgraph/types';
+import { TokensUnwrappedEntity } from '../tokens-unwrapped/tokens-unwrapped.entity';
+import { IConfig } from '../config/config.interface';
+
+@Injectable()
+export class SubgraphClientService {
+  private readonly subgraphUrl: string;
+
+  constructor(private readonly configService: ConfigService<IConfig, true>) {
+    this.subgraphUrl = this.configService.get('subgraph', { infer: true }).url;
+  }
+
+  async getTokensUnwrapped(): Promise<TokensUnwrappedEntity[]> {
+    const query = gql`
+      {
+        tokensUnwrappeds(
+          first: 5
+          orderBy: blockTimestamp
+          orderDirection: desc
+        ) {
+          id
+          from
+          targetTariAddress
+          amount
+          blockNumber
+          blockTimestamp
+          transactionHash
+        }
+      }
+    `;
+
+    try {
+      const data = await request<TokensUnwrappedsResponse>(
+        this.subgraphUrl,
+        query,
+      );
+
+      return data.tokensUnwrappeds.map((token) => {
+        const entity = new TokensUnwrappedEntity();
+        entity.id = parseInt(token.id);
```

Comment:
id: is generated by TypeOrm (PrimaryGenerateColumn), if we need to store an id from subgraph we need to add another column to `TokensUnwrappedEntity`

```suggestion
        entity.subgraphId = parseInt(token.id);
```
`